horovod/mxnet/mpi_ops.cc

```diff
@@ -203,7 +203,8 @@ int DoBroadcastCudaOnCPU(
 #endif
 
 extern "C" int horovod_mxnet_allreduce_async(NDArray* input, NDArray* output,
-                                             char* name) {
+                                             char* name,
+                                             bool average, int batch_size) {
```

Comment:
Should the last argument be number of workers instead of batch_size?

Reply:
Good catch. I think it should be num of workers. But in the example: https://github.com/ctcyang/horovod/pull/13/files#diff-e047a613860988cb493db16e369c15caL294
the gradient is scaled down by batch_size * n_workers. @ctcyang Why is that?

Reply:
It is because the default `rescale_grad` is `1.0/batch_size`. So by adding the average=True variant, the allreduce does the `1/n_workers` and the optimizer will do the `1/batch_size`.